app/models/user.rb

```diff
@@ -8,4 +9,12 @@ class User < ApplicationRecord
   validates :password,
             length: { minimum: 6 },
             if: -> { new_record? || !password.nil? }
+
+  def active?
+    active
+  end
+
+  def full_access?
+    full_access
+  end
```

Comment:
Are these needed? I thought ActiveRecord would do this automatically since these are boolean fields.

Reply:
Ohhh ... new to me Rails magic. Fixed.